packages/api/src/graphql/comment/comment.ts

```diff
@@ -208,7 +208,7 @@ export const GraphQLComment: GraphQLObjectType<Comment, Context> = new GraphQLOb
       )
     },
     revisions: {
-      type: GraphQLNonNull(GraphQLList(GraphQLNonNull(GraphQLCommentRevision)))
+      type: GraphQLList(GraphQLNonNull(GraphQLCommentRevision))
```

Comment:
revisions can simply be nullish and doesn't return any array. at least I solved the graphQL error with that change.

Reply:
The typings have revisions as never null even when empty, trying it out locally too.
When do you get revisions being null?

I think that error happens because 

```
includes: {
  revisions: true
}
```

is missing somewhere. This causes revisions not to be there

Reply:
fixed